src/excmds.ts

```diff
@@ -1879,7 +1879,31 @@ export async function loadaucmds(cmdType: "DocStart" | "DocLoad" | "DocEnd" | "T
     const aucmds = await config.getAsync("autocmds", cmdType)
     const ausites = Object.keys(aucmds)
     const aukeyarr = ausites.filter(e => window.document.location.href.search(e) >= 0)
+    const owntab = await ownTab()
+    const replacements = {
+        TRI_FIRED_MOZ_TABID: owntab.id,
+        TRI_FIRED_TRI_TABINDEX: owntab.index + 1,
+        TRI_FIRED_MOZ_WINID: owntab.windowId,
+        /* TRI_FIRED_TRI_WININDEX: owntab.windowId, */
+        TRI_FIRED_MOZ_OPENERTABID: owntab.openerTabId,
+        TRI_FIRED_ACTIVE: owntab.active,
+        TRI_FIRED_AUDIBLE: owntab.audible,
+        TRI_FIRED_MUTED: owntab.mutedInfo.muted,
+        TRI_FIRED_DISCARDED: owntab.discarded,
+        TRI_FIRED_HEIGHT: owntab.height,
+        TRI_FIRED_WIDTH: owntab.width,
+        TRI_FIRED_HIDDEN: owntab.hidden,
+        TRI_FIRED_INCOGNITO: owntab.incognito,
+        TRI_FIRED_ISARTICLE: owntab.isArticle,
+        TRI_FIRED_LASTACCESSED: owntab.lastAccessed,
+        TRI_FIRED_PINNED: owntab.pinned,
+        TRI_FIRED_TITLE: owntab.title,
+        TRI_FIRED_URL: owntab.url,
```

Comment:
I added a bunch of variables I believe might be useful to some people.

For example, TRI_FIRED_AUDIBLE which could be used for a auto muting script.